### DISCUSSION TITLE
thoughts/%ED%9A%8C%EA%B3%A0/2024-05-07/

### DISCUSSION THREAD

**Discussion by giscus** [rlaisqls/blog-comments#18] [Announcements]
# thoughts/%ED%9A%8C%EA%B3%A0/2024-05-07/

3학년 1학기가 끝나고 현장실습을 앞두고 있다. 공부도 하고, 책도 마음껏 읽고, 학교에서 이런저런 일들을 겪다보니 시간이 빠르게 지나갔다. 엄청나게 알찬 시간이었다고 할 수는 없지만 나름 여러 활동을 하며 만족스럽게 지냈다. 이제 학교에서 생활할 일이 거의 없을거라는 사실이 아쉽다.

https://blog.rlaisqls.site/thoughts/%ED%9A%8C%EA%B3%A0/2024-05-07/

**Comment by Yoochanhong** [comment]
활동을 엄청 많이 하셨네요.. 앞으로의 행보가 기대가 됩니다 ^_^

**Reply by rlaisqls** [reply]
감사합니다 :) 앞으로 더 열심히 살아보겠습니다..!
